### ISSUE TITLE
Tools not working as expected for Multiframe images.

### ISSUE THREAD

**Issue by zephyrr97531** [cornerstonejs/cornerstoneWADOImageLoader#285]
For Multiframe images, the length annotation tool displays the values in pixels instead of mm, the angle tool does not show the angle and Reference lines tool does not seem to work.
The PixelSpacing and ImageOrientationPatient values seem to coming in as undefined for these images.

**Comment by zephyrr97531** [comment]
I'm guessing this is related to the issue referenced here #194.

**Comment by swederik** [comment]
Yep, I reopened that issue because we definitely want to fix this.

As @JamesAPetts pointed out, this work was mostly done in XNAT's fork, so we can pull it back in here: https://bitbucket.org/icrimaginginformatics/ohif-viewer-xnat/src/7ce2269a6789c210b8414cef8fb8d77bcee71ac6/Packages/ohif-cornerstone-settings/imports/client/lib/classes/MetadataProvider.js#lines-347

**Comment by zephyrr97531** [comment]
How soon can this be integrated?

**Comment by swederik** [comment]
You are free to make a pull request and we will try to review it. We don't have a timeline for adding this right now. If you have funding and need this feature immediately, you might want to consider paying for [commercial support](https://docs.ohif.org/help.html#commercial-support).

Out of curiosity, what types of multiframe images are you using?

**Comment by zephyrr97531** [comment]
@swederik I'm still new to this so I'm not sure. Would the type relate to one of the DICOM properties?

**Comment by swederik** [comment]
No problem! Yeah I'm just wondering what modality (0008, 0060) they are.

**Comment by zephyrr97531** [comment]
The modality is 'MR' for the study I'm using.
